lib_static/plugins/acts_as_journalized/lib/journal_formatter/named_association.rb

```diff
@@ -53,19 +53,17 @@ def format_values(values, key, cache:)
       klass = class_from_field(key)
 
       values.map do |value|
-        if klass && value
-          record = associated_object(klass, value.to_i, cache:)
-          if record
-            if record.respond_to? :name
-              record.name
-            else
-              record.subject
```

Comment:
This extra path should no longer be necessary as [work packages have a name method](https://github.com/opf/openproject/blob/dev/app/models/work_package.rb#L343-L345) implemented by now